src/client/l10n/inputFormats.ts

```diff
@@ -30,3 +30,9 @@ export const phoneNumberFormats: Record<MarketLabel, RegExp> = {
   NO: /^(0047|\+47|47)?[2-9]\d{7}$/,
   DK: /^(0045|\+45|45)?[\d]{8}$/,
 }
+
+export const phoneNumberPlaceholder: Record<MarketLabel, string> = {
+  SE: '+46 00 000 00 00',
+  NO: '+47 000 0000',
+  DK: '+45 00000000',
+}
```

Comment:
Are we sure users can enter the phone number with spaces in it?

Reply:
I think they can't since the validation taken into place check the number of digits.
Maybe we should use a more robust validation, them insert it with or without spaces wouldn't matter.

Reply:
hmm. you are right. I will remove the character max length value so the user can add spaces. 
A more robust validation + error handling will come for all fields but we need the phone number in NO checkout quickly.